### PR TITLE
update landing page

### PR DESCRIPTION
**Pull request by oindrillac**
Reduced textual content on landing page keeping it very concise.

Please let me know your thoughts.

### CODE REVIEW

**Comment by oindrillac** [comment]
image

File: content/index.md [hunk 1]
```diff
@@ -2,80 +2,33 @@
...
-To learn about Open Data Hub and its architecture, visit [opendatahub.io](https://www.opendatahub.io).
-
-To get started with using ODH applications deployed and running on an open cloud instance, visit the **MOC - ODH Users** section.
-
-To get started with deploying components on ODH, visit the **MOC - ODH Operations** section.
-
-To learn more about Operate First: making cloud operations as fundamental as functionality in the upstreams. Read the [Operate First Community Manifesto](https://openinfralabs.org/operate-first-manifesto/)
-
+ 
+Open Data Hub (ODH) is a data and AI platform for the hybrid cloud running on on Red Hat's [Kubernetes](https://kubernetes.io/)-based [OpenShift® Container Platform](https://www.openshift.com/) and [Ceph Object Storage](https://www.redhat.com/en/technologies/storage/ceph).
```
Comment (durandom) [inline content/index.md]: remove this sentence. It's the same as on the ODH landing page :)

File: content/index.md [hunk 2]
```diff
@@ -2,80 +2,33 @@
...
-
-As one of the first steps, we have begun operating Open Data Hub on the Mass Open Cloud(MOC) in an open cloud environment, before we ship it to our customers. At the AICoE, we are focused on creating examples of how ODH is operated and deployed in an open cloud environment, how we perform open source data science in an open cloud environment and sharing our learnings with the community.
-
-This website acts as a landing site for sharing examples from our experience of operating Open Data Hub in an open cloud environment. It is targeted to serve as an upstream platform where a wider community can participate and leverage our work (and we theirs), ultimately to drive an open source solution for cloud operation.
-
+ 
+[Operate First](https://openinfralabs.org/operate-first-manifesto/) is an initiative focused on adding operational intelligence into development of software by operating them in a transparent, open cloud environment.
```
Comment (durandom) [inline content/index.md]: I think we need some more sentences that describe Op1st. Let's replace this sentence with:

```
[Operate First](https://openinfralabs.org/operate-first-manifesto/) is an initiative to operate software in a production-grade environment - bringing users, developers and operators closer together.

The goal is to create an Open Cloud environment, with reproducibility built-in, operated by a Community.

Open means, onboarding and getting involved should mimic the process of an Open Source project, where planning, issue tracking and the code are accessible in a read-only fashion.

Reproducibility caters towards being a blueprint for other setups. If we don’t want each environment to be a snowflake, we should be able to extract best practices that are easy to apply to new environments.
```

File: content/index.md [hunk 3]
```diff
@@ -2,80 +2,33 @@
...
-To get started with deploying components on ODH, visit the **MOC - ODH Operations** section.
-
-To learn more about Operate First: making cloud operations as fundamental as functionality in the upstreams. Read the [Operate First Community Manifesto](https://openinfralabs.org/operate-first-manifesto/)
-
+ 
+Open Data Hub (ODH) is a data and AI platform for the hybrid cloud running on on Red Hat's [Kubernetes](https://kubernetes.io/)-based [OpenShift® Container Platform](https://www.openshift.com/) and [Ceph Object Storage](https://www.redhat.com/en/technologies/storage/ceph).
+ 
+## [Data Science](https://www.operate-first.cloud/data-science/)
+ 
+Get started with tutorials and examples for data science on ODH.
```
Comment (durandom) [inline content/index.md]: s/ODH/Open Data Hub/

File: content/index.md [hunk 4]
```diff
@@ -2,80 +2,33 @@
...
+ 
+## [Users](https://www.operate-first.cloud/users/)
+ 
+Learn how you can engage with ODH and access the deployed components.
```
Comment (durandom) [inline content/index.md]: s/ODH/Open Data Hub/

File: content/index.md [hunk 5]
```diff
@@ -2,80 +2,33 @@
...
+ 
+## [Operators](https://www.operate-first.cloud/operators/)
+ 
+Get started with deploying various components on ODH and find information on operating it.
```
Comment (durandom) [inline content/index.md]: See how we are deploying and operating Open Data Hub

File: content/index.md [hunk 6]
```diff
@@ -2,80 +2,33 @@
...
+ 
+## [Blueprints](https://www.operate-first.cloud/blueprints/)
+ 
+Learn about repeatable best practices that can be applied to other projects.
```
Comment (durandom) [inline content/index.md]: Apply best practices and tooling to your own projects.

**Review by durandom — CHANGES_REQUESTED** [review]
good start, thanks.

I've added some tweaks.

**Comment by oindrillac** [comment]
@durandom thanks for your inputs :+1:  Please see the changes

**Review by durandom — APPROVED** [review]
🌮

**Pull request merged**